lee_harvey/class_scripts/temperature.py

```diff
@@ -0,0 +1,36 @@
+# Defined function to convert celsius to fahrenheit
+def convert_to_fahrenheit(v):
+    return v * 1.8 + 32
+
+# Defined function to convert fahrenheit to celsius
+def convert_to_celsius(x):
+    return (x - 32) * .5556
+
+# Provides the celsius of 100 degrees fahrenheit
+celsius_100 = convert_to_fahrenheit(100)
+# Provides the fahrenheit of 0 degrees celsius
+celsius_0 = convert_to_celsius(0)
+# Define variable degrees_c
+degrees_c = 30.2
+# Define variable degrees_f
+degrees_f = 85.1
+
+# Display the 100 celsius conversion
+print(str(celsius_100) + " fahrenheit")
+# Display the 0 fahrenheit conversion
+print(str(round(celsius_0))+ " celsius")
+# Display the 34.2 fahrenheit conversion
+print(str(convert_to_celsius(34.2)) + " celsius")
```

Comment:
please make sure to document code better. For example, if these are the answers to the challenge indicate in your comments that is the answer to questions 1-3.